modules/installation-identify-supported-aws-outposts-instance-types.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// installing/installing_aws/installing-aws-outposts-remote-workers.adoc
+
+:_content-type: PROCEDURE
+[id="installation-identify-supported-aws-outposts-instance-types_{context}"]
+= Identifying your AWS Outposts instance types
+
+AWS Outposts rack catalog includes options supporting the latest generation Intel powered EC2 instance types with or without local instance storage.
+Identify which instance types are configured in your AWS Outpost instance. As part of the installation process, you must update the `install-config.yaml` file with the instance type that the installation program will use to deploy worker nodes.
+
+.Procedure
+
+Use the AWS CLI to get the list of supported instance types by running the following command:
+[source,terminal]
+----
+$ aws outposts get-outpost-instance-types --outpost-id <outpost_id> <1>
+----
+<1> For `<outpost_id>`, specify the Outpost ID, used in the AWS account for the worker instances
+
++
+[IMPORTANT]
+====
+When you purchase capacity for your AWS Outpost instance, you specify an EC2 capacity layout that each server provides. Each server supports a single family of instance types. A layout can offer a single instance type or multiple instance types. Dedicated Hosts allows you to alter whatever you chose for that initial layout. If you allocate a host to support a single instance type for the entire capacity, you can only launch a single instance type from that host.
```

Comment:
```suggestion
When you purchase capacity for your AWS Outpost instance, you specify an EC2 capacity layout that each server provides. Each server supports a single family of instance types. A layout can offer a single instance type or multiple instance types. Dedicated Hosts allows you to alter whatever you chose for that initial layout. If you allocate a host to support a single instance type for the entire capacity, you can only start a single instance type from that host.
```

Reply:
IBMSG p 358 advises to use "start" instead of launch

Reply:
Thanks @skrthomas I've applied your feedback.